core/token_auth.go

```diff
@@ -0,0 +1,93 @@
+package core
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+)
+
+func (c *Core) getWholeTokenValue(wholeTokenName string) (int, string) {
```

Comment:
Add this as separate package outside core, like RAC create separate package token and add this files,

Reply:
moved to a separate package called token